packages/swap/src/event-handlers/swapEgressScheduled.ts

```diff
@@ -129,7 +135,7 @@ export default async function swapEgressScheduled({
     });
   } else {
     egressFee = bigintMin(
-      await getEgressFeeAtBlock(block.hash, swap.destAsset),
+      await getEgressFeeAtBlock(block.hash, block.specId, swap.destAsset),
```

Comment:
i think we dont need to handle this case because with spec 120 the fee should be on the event

Reply:
if its not on the event, the rpc will return the old format

Reply:
ok interesting i didn't realize that was the case.. based on the ticket you wrote.. it sounded like we were to read this value from the rpc 

https://linear.app/chainflip/issue/WEB-861/adjust-quoted-ingressegress-fees-to-protocol-changes

Reply:
yeah i was not sure at that time how the fees will be exposed from the events 🙂 but we can leave the change, it should not hurt

Reply:
okay i fixed it. think doesn't make sense to handle it when the event change already does it 👍